source/agora/node/Ledger.d

```diff
@@ -772,60 +772,52 @@ public class Ledger
         const Scalar challenge = hashFull(block);
         const enrolled_validators = this.enroll_man.getCountOfValidators(block.header.height);
 
-        log.trace("Validators expected to sign this block is {}", enrolled_validators);
         // Check that more than half have signed
         auto signed = iota(0, enrolled_validators).filter!(i => block.header.validators[i]).count();
         if (signed <= enrolled_validators / 2)
         {
-            log.error("Only {} signed. Require more than {} out of {} validators to sign for externalizing slot height {}.",
-                      signed, enrolled_validators / 2, enrolled_validators, block.header.height);
-            return "Only " ~ to!string(signed) ~ " signed the block. Require more than " ~ to!string(enrolled_validators);
+            // Can't use '51%' here because 50.000001% is enough
+            log.error("Block#{}: Signatures are not majority: {}/{} ({})",
```

Comment:
A little cryptic but I guess the intention is to keep it more compact?

Reply:
Is it cryptic ? It'll show something like:
```
Block#4: Signatures are not majority: 2/6 ([12])
```
The `[12]` is very cryptic tho and will need to be fixed, but I thought "majority" and "2/6" will suffice. Suggestions welcome.

Reply:
Maybe if we add `Block#4: Signatures are not majority: 2/6, validators signed=[252]`.
We could also show the bits of the mask but then if we have hundreds of validators it may get messy.

Reply:
How does it look now ?